apps/remix-ide-e2e/src/helpers/hardhat_compilation.ts

```diff
@@ -0,0 +1 @@
+// @ts-ignore
```

Comment:
why is this file needed?

Reply:
this is the artifact generated by hardhat, this emulate hardhat in e2e tests